plugins/modules/ndo_route_map_policy_multicast.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_route_map_policy_multicast
+short_description: Manage Multicast Route Map Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Multicast Route Map Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+author:
+- Akini Ross (@akinross)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a tenant template.
+    type: str
+    required: true
+  route_map_policy:
+    description:
+    - The name of the Multicast Route Map Policy.
+    type: str
+    aliases: [ name ]
+  route_map_policy_uuid:
+    description:
+    - The uuid of the Multicast Route Map Policy.
+    - This parameter is required when the O(route_map_policy) needs to be updated.
+    type: str
+    aliases: [ uuid ]
+  description:
+    description:
+    - The description of the Multicast Route Map Policy.
+    type: str
+  entries:
+    description:
+    - A list of entries attached to the Multicast Route Map Policy.
+    - The list of configured entries must contain at least one entry.
+    - When the list of entries is null the update will not change existing entry configuration.
+    type: list
+    elements: dict
+    suboptions:
+      order:
+        description:
+        - The order of the entry.
+        type: int
+        required: true
+      group:
+        description:
+        - The name of the entry.
+        type: str
+        required: true
+      source:
+        description:
+        - The data of the entry.
+        type: str
+        required: true
+      rp:
+        description:
+        - The rendezvous point of the entry.
+        type: str
+        aliases: [ rendezvous_point ]
```

Comment:
I think the name and the alias should be switched.
```suggestion
      rendezvous_point:
        description:
        - The rendezvous point of the entry.
        type: str
        aliases: [ rp ]
```

Reply:
why?

Reply:
As a general option convention, If I was adding the full name as an option/alias, I would use shorter names and alternatives in alias. However, it is up to you as there is no discernible benefit either way.

Reply:
ok will keep as is